packages/cozy-harvest-lib/src/helpers/oauth.js

```diff
@@ -75,6 +75,20 @@ export const handleOAuthResponse = () => {
   return true
 }
 
+/**
+ * Generate oauth popup url
+ * @param  {string} cozyUrl cozy url
+ * @param  {string} accountType connector slug
+ * @param  {string} oAuthStateKey localStorage key
+ * @param  {Object} oAuthConf connector manifest oauth configuration
+ * @param  {string} nonce unique nonce string
+ */
+export const getOAuthUrl = (cozyUrl, accountType, oAuthStateKey, oAuthConf, nonce) => {
+  let oAuthUrl = `${cozyUrl}/accounts/${accountType}/start?state=${oAuthStateKey}&nonce=${nonce}`
+  if (oAuthConf.scope) oAuthUrl+= `&scope=${oAuthConf.scope}`
```

Comment:
Some servers will require the `scope` parameter, even if empty. 

The real fix should probably be on the Enedis side (The client is allowed by the spec to send a scope parameter).

If you want a workaround here, you should at least make a difference between an empty list / empty string, and a null / undefined / false value. If you can't, you'd better define some magic value that means "do not send a scope parameter"

Reply:
I think adding a scope : false / undefined can be a good compromise here :)

Reply:
Thanks, I will choose undefined

Reply:
I finally chose an empty array as "no scope" value to have the most possibilities available